sdk/core/azure-core/CHANGELOG.md

```diff
@@ -8,6 +8,14 @@
 
 ### Bugs Fixed
 
+- [[#4206]](https://github.com/Azure/azure-sdk-for-cpp/issues/4206) Azurite parallel connectivity problem.
```

Comment:
From what I understand, this is not an issue with Azurite, but instead a general problem which occurs whenever a server terminates the TCP connection during the HTTP exchange.

Reply:
Maybe "Fixed connectivity issues which can occur if a TCP connection is dropped prematurely"?

Reply:
@LarryOsterman, #4262